Readme.md

```diff
@@ -1,36 +1,48 @@
 <!-- default badges list -->
-![](https://img.shields.io/endpoint?url=https://codecentral.devexpress.com/api/v1/VersionRange/128589875/23.1.6%2B)
 [![](https://img.shields.io/badge/Open_in_DevExpress_Support_Center-FF7200?style=flat-square&logo=DevExpress&logoColor=white)](https://supportcenter.devexpress.com/ticket/details/T381904)
 [![](https://img.shields.io/badge/📖_How_to_use_DevExpress_Examples-e9f6fc?style=flat-square)](https://docs.devexpress.com/GeneralInformation/403183)
 <!-- default badges end -->
-<!-- default file list -->
-*Files to look at*:
 
-* [OrdersChart.ascx](CS/WebChart/WebChart.Web/OrdersChart.ascx) 
-* **[OrdersChart.ascx.cs](CS/WebChart/WebChart.Web/OrdersChart.ascx.cs)**
-* [orders-chart.js](CS/WebChart/WebChart.Web/Scripts/Controls/orders-chart.js)
-<!-- default file list end -->
-# How to display the dxChart widget in an XAF view
+# How to display the dxChart widget in an XAF ASP.NET WebForms view
+
+This example illustrates how to show a chart control with a lot of points. The built-in [Charts module](https://docs.devexpress.com/eXpressAppFramework/113302/analytics/chart-module) draws all points on the same screen at once, and this can be inconvenient for an end user. To improve usability, it is possible to implement real-time zooming and scrolling. The `dxChart` widget from the DevExtreme library fits this scenario perfectly. This example shows how to use this widget in an XAF application.
```

Comment:
```suggestion
This example illustrates how to show a chart control with many points. The built-in [Charts module](https://docs.devexpress.com/eXpressAppFramework/113302/analytics/chart-module) draws all points on the same screen at once, and this can affect readability. To improve usability, you can implement real-time zooming and scrolling, for example, the `dxChart` widget from the DevExtreme library. This example shows how to use this widget in an XAF application.
```